src/games/strategy/engine/framework/GameDataManager.java

```diff
@@ -180,7 +181,7 @@ private void updateDataToBeCompatibleWithNewEngine(final Version originalEngineV
      * }
      * } catch (final Exception e)
      * {
-     * e.printStackTrace();
+     * ClientLogger.logQuietly(e);
```

Comment:
Look at the context surrounding it. We may want to axe the entire method. For the moment leaving it is fine.